logarithm/index.html

```diff
@@ -219,7 +219,7 @@ <h3 style="display: inline;">example username</h3>
         instanceElement.href = instanceConfig?.frontend
 
         const timeElement = post.querySelector('time')
-        timeElement.innerText = getRelativeTime(new Date(+date - 18000000))
```

Comment:
I think you may be confused by my lack of documentation but this code is actually what I used to fix the 5 hour time difference (the date variable is off). That being said, Svenlaa made the discovery that this is a time zone issue, so depending on your time zone this does not fix the issue. I'll be fixing this in my repo soon, and can probably PR yours to update the fix to be more accurate and less hard-coded

Reply:
Ahh i see... apologies

Reply:
All good (: